GenderPayGap.WebUI/Controllers/ManageOrganisations/ManageOrganisationsController.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Core.Models.HttpResultModels;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.WebUI.Classes;
+using GenderPayGap.WebUI.ErrorHandling;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Organisation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GenderPayGap.WebUI.Controllers.ManageOrganisations
+{
+    [Authorize(Roles = LoginRoles.GpgEmployer)]
+    [Route("manage-organisations-new")]
+    public class ManageOrganisationsController : Controller
+    {
+
+        private readonly IDataRepository dataRepository;
+
+        public ManageOrganisationsController(IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+        [HttpGet]
+        public IActionResult ManageOrganisationsGet()
+        {
+            User user = ControllerHelper.GetGpgUserFromAspNetUser(User, dataRepository);
+            
+            // Show the privacy policy to non-admin users (if they're not being impersonated) if they haven't read it yet

```

Comment:
As far as I can tell, this isn't used elsewhere at the moment - although it seems slightly odd to me that we'd only show the privacy policy here and not elsewhere? 

@jamesgriff do you think this would be useful?

Reply:
I like it! I agree with Dulcie, it does feel odd we only show the privacy policy here - but I think that's a problem with the business logic not with the code, so pulling it out feels right to me

Reply:
Clever!
Is it semantically "good practice" for an exception to redirect to an actual page rather than an error page of some kind? Not questioning it, just curious!

Reply:
Well I sort of like "exception as fall-through behaviour" - notwithstanding it's sort of the only way we'd do this particular thing - an exception shouldn't just be "something has gone wrong" but "something has happened that's an exception to the usual rules of what is going on" - no reason that behaviour should be pathological. This is quite similar to "redirect to a signin page if you're not logged in", certainly it's a pattern we use across BBC Monitoring (where the permissions system is multi-layered)